### PR TITLE
feat: add azure-identity in every test

### PR DESCRIPTION
**Pull request by ChenTanyi**

### CODE REVIEW

File: sdk/appservice/mgmt/pom.xml [hunk 1]
```diff
@@ -87,6 +87,16 @@
...
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-http-netty</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-identity</artifactId>
```
Comment (xccc-msft) [inline sdk/appservice/mgmt/pom.xml:97]: Why we need add azure-identity in test?

Comment (ChenTanyi) [reply sdk/appservice/mgmt/pom.xml:97]: I have removed it from compile scope, so it needs to be added in test scope.

Comment (xccc-msft) [reply sdk/appservice/mgmt/pom.xml:97]: Got it.

**Comment by ChenTanyi** [comment]
storage test is nothing to do with management change

**Pull request merged**